docs/platforms/javascript/common/configuration/integrations/index.mdx

```diff
@@ -15,9 +15,9 @@ However, it's important to note that not all integrations are compatible with al
 
 Depending on whether an integration enhances the functionality of a particular runtime, such as the BrowserTracing integration for the browser runtime or the RequestData integration for the Node.js runtime, you can only include these integrations in their respective configuration files:
 
-- For the browser runtime, add integrations to `sentry.client.config.(js|ts)`.
-- For Node.js, add integrations to your Sentry setup in `instrumentation.(js|ts)`.
```

Comment:
just to clarify for me, I thought the instrumentation.js file was now required? Is this not the case? or is this something else?

Reply:
The instrumentation file is indeed required, but we still suggest people add all the `sentry.*.config.ts` files for the simple reason that if you want to add a integration that is node specific, and you import it from `instrumentation.ts`, you may run into problems because Node.js apis are not available in edge and instrumentation.ts is ran for both. Having the `sentry.*.config.ts` files mitigates that issue.